lib/zendesk_apps_support/validations/manifest.rb

```diff
@@ -110,6 +116,33 @@ def oauth_cannot_be_secure(manifest)
           end
         end
 
+        def invalid_secure_param_scopes_error(manifest)
+          errors = []
+          
+          manifest.parameters.each do |parameter|
+            next if parameter.scopes.nil?
+
+            unless parameter.secure
+              errors << ValidationError.new(:scope_requires_secure_parameter)
+              next 
+            end
+
+            unless parameter.scopes.any?
```

Comment:
Would `if parameters.scopes.empty` work here? I find `unless` a bit harder to read compared to `if`.